test/state-colors.json

```diff
@@ -1,8 +0,0 @@
-{
-  "error": "#E65140",
-  "info": "#0582DA",
-  "success": "#00BF68",
-  "warning": "#FFBE00",
-  "highlighted": "#FFF8E5",
-  "notspecified": "#CCCCCC"
-}
```

Comment:
I don't believe this file is being used by anything. Even before this PR.

And the `error` color is incorrect in this file as well.